### PR TITLE
Bumps to 0.8.5

### PR DESCRIPTION
**Pull request by damncabbage**
I'm going off the version bump made in b33995f01500efcf98fbf1b06de8ba344bed88d9; please forgive me if I've missed something. :sweat_smile:

### CODE REVIEW

**Comment by damncabbage** [comment]
```
<damncabbage> Uh oh. Build terminated?
              https://travis-ci.org/purescript-contrib/node-purescript-bin/builds/124696488
<damncabbage> Does someone have to pre-build, or am I just unlucky?
```

**Comment by garyb** [comment]
Build timed out again, I'm giving `travis_wait` a try [here](https://github.com/purescript-contrib/node-purescript-bin/commit/6dab9dda0851a1da4885a090027f67623a462d24) to see if it can work around the long compile.

**Comment by damncabbage** [comment]
@shinnn: Can you explain what you're doing here? https://github.com/purescript-contrib/node-purescript-bin/commit/93c845c90dc3979b7e7e88e06e10d0cbce8df45d

**Comment by shinnn** [comment]
@damncabbage Never mind. Just an experimental commit and it doesn't have any good effects.

**Comment by damncabbage** [comment]
yaaaaaaay

**Comment by damncabbage** [comment]
:tada: (Thanks.)

**Comment by garyb** [comment]
Thanks for the update! Just testing locally also, and then will publish.

**Comment by shinnn** [comment]
@garyb Why squashed. Those two commits are for totally different purpose.

**Comment by garyb** [comment]
Sorry, force of habit.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "purescript",
-  "version": "0.8.4",
+  "version": "0.8.5",
```
Comment (shinnn) [inline package.json]: https://twitter.com/sindresorhus/status/298376330544824320

Comment (damncabbage) [inline package.json]: I wouldn't usually do this, but the reason for this PR is to, well, bump the version. I followed the 0.8.4 and 0.8.3 commits.

**Pull request merged**